docs/10_utilities/fill-queue/fill-queue-how-to.md

```diff
@@ -0,0 +1,100 @@
+---
+content_title: How-To Setup And Use Fill-Queue
+link_text: How To Setup and Use Fill-Queue
+---
+
+## Summary
+
+This how-to demonstrates the set up and use of [Fill-Queue application](index) running locally.
+
+## Prerequisites
+
+Before you begin, make sure you have the following information or have completed the following referenced procedures:
+
+1. `nodeos` runs locally.
```

Comment:
there is no requirement to run nodeos locally. you can connect to it via network